pyam/validation.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import pandas as pd
+
+from pyam.utils import make_index, s
```

Comment:
I'd suggest renaming the function `s` into something more descriptive.

Reply:
Maybe `plural_s`? Although I typed that before I'd seen the use in a docstring. There it's quite readable, so ok from my side to keep it this way if we don't have any good ideas.